app/src/main/java/fr/free/nrw/commons/explore/SearchActivity.java

```diff
@@ -54,12 +57,20 @@
     ViewPagerAdapter viewPagerAdapter;
     private String query;
 
+    @RequiresApi(api = Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
The app would crash on devices with lower android versions. Can you use a different method for setting background? :)